src/App.js

```diff
@@ -88,15 +103,8 @@ function App() {
         const dateTimestamp = Date.now();
         const curDate = new Date(dateTimestamp);
         setPsiturk(new PsiTurk(turkId, "/complete"));
-        // study id yet to be determined
-        jsPsych.data.addProperties({
-          participant_id: turkId,
-          study_id: "mturk",
-          timestamp: dateTimestamp.toString(),
-          start_date: curDate.toString(),
-        });
         setMethod("mturk");
-        setLogin(true);
+        setLoggedIn(true, "mturk", turkId, startDate, timestamp)
```

Comment:
why do the startDate and timestamp need to be passed if they're part of the state of the callback? (also, just curious)